CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * The `:` revset operator is deprecated. Use `::` instead.
 
+* The `--allow-large-revsets` flag for `jj rebase` and `jj new` can no longer be
```

Comment:
(Non-blocking, re commit number 3) I don't remember, why are we prohibiting duplicate destinations? This affects only `-d`, not `-s`, correct?

Another option would be to warn the user about duplicate revisions and let the undo.

At this point, you've thought this through much more deeply than I have, and as you point out we can always change our mind later, so this is not blocking. I can always think about it later.

Reply:
It does affect `-s` and `-b` too. I also don't remember why :)